Lectures/Lecture2_GitforCollaboration.md

```diff
@@ -29,10 +29,14 @@ The (physics) research community is the main community that should be considered
 + Readme, templates, community rules, licensing and credit
 
 ## Evaluation
-The students will go through lessons 6-13 of the [git tutorial](https://git.science.uu.nl/os4p-2023/git-tutorial) and will be asked to complete an exercise to demonstrate their understanding of the material.
+During the four hours of tutorial, the students will go through the fourteen lessons of the [git tutorial](https://os4p.gitbook.io/git-tutorial/) and will be asked to complete an exercise to demonstrate their understanding of the material.
 
 ### Assignment description
-Students will be grouped in threes and asked to create a repository on GitLab and to use it to collaborate with the two others on a small project where they write a [haiku poem](https://en.wikipedia.org/wiki/Haiku) about Open Science, each contributing one line.
+Students will form teams of threes and asked to create a branch on GitLab and to use it to collaborate with the two others on a small project where they write a [haiku poem](https://en.wikipedia.org/wiki/Haiku) about Open Science, each contributing one line from their own branch. They then create a Merge Request to merge the file with the haiku into the master branch. The students will use the features of GitLab to manage this assignment.
```

Comment:
can you be more explicit in the instructions? will the team of 3 work on the same branch but separate computers? does it mean all need to have their own clones? 
please specify "They" and "then" in the penultimate sentence.